ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/RDXVisualizer.java

```diff
@@ -26,11 +32,31 @@ public abstract class RDXVisualizer implements RDXRenderableProvider
    private Consumer<Boolean> activenessChangeCallback;
    private boolean wasActive = false;
 
+   @Nullable
+   private ROS2Heartbeat heartbeat;
+
    public RDXVisualizer(String title)
    {
       this.title = ImGuiTools.uniqueLabel(title);
```

Comment:
I would change this to
```suggestion
      this.title = title;
```
because you've already got `if (ImGui.collapsingHeader(labels.get(title)))` below.

Reply:
Changed